utils/src/test/kotlin/ArchiveUtilsTest.kt

```diff
@@ -76,5 +79,29 @@ class ArchiveUtilsTest : StringSpec() {
             fileB.exists() shouldBe true
             fileB.readText() shouldBe "b\n"
         }
+
+        "Debian deb archive can be unpacked" {
+            val tempDir = createTestTempDir(ORT_NAME)
+            val archiveDeb = File("src/test/assets/testpkg.deb")
+            val archiveUdepTemp = tempDir.resolve("testpkg.udeb")
+            val archiveUdep = archiveDeb.copyTo(archiveUdepTemp)
+
+            listOf(archiveDeb, archiveUdep).forEach { archive ->
```

Comment:
Please use `forAll` instead of `forEach` for better output in case of assertion failure.